docs/book/component-gallery/orchestrators/sagemaker.md

```diff
@@ -0,0 +1,121 @@
+---
+description: How to orchestrate pipelines with Amazon Sagemaker
+---
+
+The Sagemaker orchestrator is an [orchestrator](./orchestrators.md) flavor provided
+with the ZenML `aws` integration that uses [Amazon Sagemaker](https://aws.amazon.com/sagemaker)
+to run your pipelines.
+
+{% hint style="warning" %}
+This component is only meant to be used within the context of [remote ZenML deployment scenario](../../getting-started/deploying-zenml/deploying-zenml.md). Usage with a local ZenML deployment may lead to unexpected behavior!
+{% endhint %}
+
+## When to use it
+
+You should use the Sagemaker orchestrator if:
+* you're already using AWS.
+* you're looking for a proven production-grade orchestrator.
+* you're looking for a UI in which you can track your pipeline runs.
+* you're looking for a managed solution for running your pipelines.
+* you're looking for a serverless solution for running your pipelines.
+
+## How to deploy it
+
+In order to use a Sagemaker AI orchestrator, you need to first deploy [ZenML to the
+cloud](../../getting-started/deploying-zenml/deploying-zenml.md). It would be
+recommended to deploy ZenML in the same region as you plan on using for
+Sagemaker, but it is not necessary to do so. You must ensure that you are
+[connected to the remote ZenML
+server](../../starter-guide/collaborate/zenml-deployment.md) before using this
+stack component.
+
+The only other thing necessary to use the ZenML Sagemaker orchestrator is
+enabling the relevant permissions for your particular role.
+
+In order to quickly enable APIs, and create other resources necessary for to use
+this integration, we will soon provide a Sagemaker stack
+recipe via [our `mlops-stacks` recipe repository](https://github.com/zenml-io/mlops-stacks), which
+will help you set up the infrastructure with one click.
+
+## How to use it
+
+To use the Sagemaker orchestrator, we need:
+
+* The ZenML `aws` integration installed. If you haven't done so, run 
+    ```shell
+    zenml integration install aws
+    ```
+* [Docker](https://www.docker.com) installed and running.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) installed.
+* A [remote artifact store](../artifact-stores/artifact-stores.md) as part of 
+your stack (configured with an `authentication_secret` attribute) along with [a remote secrets manager](../secrets-managers/secrets-managers.md) to store that
+authentication secret.
+* A [remote container registry](../container-registries/container-registries.md) 
+as part of your stack.
+* An IAM role or user with an `AmazonSageMakerFullAccess` policy applied to it
+  as well as `sagemaker.amazonaws.com` added as a Principal Service. Full
+  details on these permissions can be found
+  [here](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html)
+  or use the ZenML recipe (when available) which will set up the necessary
```

Comment:
Correct me if im wrong, but doesnt the role also need access to the container registry, secrets manager, and the artifact store? How else would it be interacting with these things? Maybe mention here that you need to add the neccessary permissions to the sagemaker role

Reply:
All (apart from secrets manager access) are all included with the `AmazonSageMakerFullAccess`. I tested this in a fresh account yesterday.